index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
```

Comment:
We already have the `baseLatency` in the context, this needs to be moved to there. Getting two different latency values from different entities is weird. Also this should be renamed - I prefer `deviceOutputLatency`.